types/artifacts.d.ts

```diff
@@ -409,10 +409,26 @@ declare global {
         attributeWidth: string;
         /** The raw height attribute of the image element. CSS images will be set to the empty string. */
         attributeHeight: string;
-        /** The CSS width property of the image element. */
+        /**
+         * The CSS width property of the image element.
+         * @deprecated Use `element?._privateCssSizing.width` instead.
+         */
         cssWidth?: string;
-        /** The CSS height property of the image element. */
+        /**
+         * The CSS height property of the image element.
+         * @deprecated Use `element?._privateCssSizing.width` instead
+         */
         cssHeight?: string;
+        /**
+         * The width/height of the element as defined by matching CSS rules. Set to `undefined` if the data was not collected.
+         * @private
```

Comment:
seems not worth it to `ts-expect-error` all uses of this to keep it `@private`? We could actually mark this `@deprecated` which is kind of backwards but would discourage use and strike it out in vscode but still leave it visible for type checking. Maybe there are other options?

Reply:
i dig it. done. and added some careful wording around it.